### DISCUSSION TITLE
NextJS 13 using api folder for Python runtime routes

### DISCUSSION THREAD

**Discussion by SilenNaihin** [vercel/community#2732] [Bug Report]
### Summary

Getting errors posting in the help forum so posting here. Having issues with Python runtime locally vs on Vercel. Would love some help/suggestions!

pip show openai is defined everywhere locally, but opening the fil in api/completions/utils/llm_response.py it gives me a Import "openai" could not be resolved error. When I run through vercel.dev with requirements.txt containing openai at my root, everything works fine on localhost, no issue.
```
However, in production on Vercel, the routes work so the api loads the functions correctly, but when the index.py file calls the llm_response.py file I'm getting the error below:
stdout: b''
stderr: b'Traceback (most recent call last):\n  File "/var/task/api/transcribe/whisper.py", line 1, in <module>\n    import openai\nModuleNotFoundError: No module named 'openai'\n'
stdout: b''
stderr: b'Traceback (most recent call last):\n  File "/var/task/api/transcribe/whisper.py", line 1, in <module>\n    import openai\nModuleNotFoundError: No module named 'openai'\n'
```

This is the api/completions/utils/llm_response.py file that's being called by api/completions/index.py.
 ```
import openai
import os

...

os.getenv("OPENAI_API_KEY")
OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")

...
```

This is the repo I am working in:
https://github.com/SilenNaihin/roadman-talk

### Example

https://roadman-talk.vercel.app/

### Steps to Reproduce

`git clone` the repo I shared. then `npm i` for all the modules. Then vercel dev to run the Vercel cli - this works. Add your env variables at .env.local from the .env.template. Upload to Vercel and try to run any of the api calls

**Comment by DirkHierold** [comment]
how did you solved it? ihave a similiar problem with vercel not founding my openai module
